### ISSUE TITLE
Various scripts installed under `/usr/share/doc/`

### ISSUE THREAD

**Issue by holmanb** [neomutt/neomutt#3444]
## Expected Behaviour
scripts should be installed separate from docs

## Actual Behaviour
```
make install
mkdir -p po
mkdir -p docs
installing /usr/share/locale/bg/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/ca/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/cs/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/da/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/de/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/el/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/en_GB/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/eo/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/es/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/et/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/eu/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/fi/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/fr/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/ga/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/gl/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/hu/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/id/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/it/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/ja/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/ko/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/lt/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/nb_NO/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/nl/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/pl/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/pt_BR/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/ru/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/sk/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/sr/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/sv/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/tr/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/uk/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/zh_CN/LC_MESSAGES/neomutt.mo
installing /usr/share/locale/zh_TW/LC_MESSAGES/neomutt.mo
mkdir -p /etc
install -c -m 644 docs/neomuttrc /etc/neomuttrc
for d in colorschemes logo oauth2 vim-keys account-command/macos-keychain account-command/gpg-json; do \
	echo "Creating directory /usr/share/doc/neomutt/$d"; \
	install -c -d -m 755 /usr/share/doc/neomutt/$d || exit 1; \
	for f in $(find ./contrib/$d -type f); do \
		echo "Installing $f"; \
		install -c -m 644 $f /usr/share/doc/neomutt/$d || exit 1; \
	done \
done
Creating directory /usr/share/doc/neomutt/colorschemes
Installing ./contrib/colorschemes/neonwolf-256.neomuttrc
Installing ./contrib/colorschemes/solarized-dark-256.neomuttrc
Installing ./contrib/colorschemes/vombatidae.neomuttrc
Installing ./contrib/colorschemes/zenburn.neomuttrc
Creating directory /usr/share/doc/neomutt/logo
Installing ./contrib/logo/neomutt-128.png
Installing ./contrib/logo/neomutt-256.png
Installing ./contrib/logo/neomutt-32.png
Installing ./contrib/logo/neomutt-64.png
Installing ./contrib/logo/neomutt.svg
Creating directory /usr/share/doc/neomutt/oauth2
Installing ./contrib/oauth2/mutt_oauth2.py
Installing ./contrib/oauth2/mutt_oauth2.py.README
Creating directory /usr/share/doc/neomutt/vim-keys
Installing ./contrib/vim-keys/README.md
Installing ./contrib/vim-keys/vim-keys.rc
Creating directory /usr/share/doc/neomutt/account-command/macos-keychain
Installing ./contrib/account-command/macos-keychain/README.md
Installing ./contrib/account-command/macos-keychain/keychain.py
Creating directory /usr/share/doc/neomutt/account-command/gpg-json
Installing ./contrib/account-command/gpg-json/README.md
Installing ./contrib/account-command/gpg-json/credentials.sh
find /usr/share/doc/neomutt \( -name "*.sh" -o -name "*.py" \) -exec chmod a+x {} \;
# Install binaries
mkdir -p /usr/bin
for f in neomutt; do \
	install -c $f /usr/bin/`basename $f` || exit 1; \
done
mkdir -p /usr/libexec/neomutt
for f in pgpewrap ./contrib/smime_keys; do \
	install -c $f /usr/libexec/neomutt/`basename $f` || exit 1; \
done
```

I [reported to one downstream distro](https://bugs.gentoo.org/853241) as well, more (including debian/ubuntu) appear affected as well.

**Comment by gahr** [comment]
I'm not sure I'd tag this as a bug.. but we have ideas how to fix this, see #3431. I'd probably close this as a dup of that.

**Comment by flatcap** [comment]
Noted :-)
